provider/source/data/debug/timezone/periods/v1.json

```diff
@@ -0,0 +1,4734 @@
+      48
+    ],
+    "1981-05-01 00:00": [
+      "+1:00",
+      46
+    ]
+  },
+  "ecgps": {
+    "1970-01-01 00:00": [
+      "-5:00",
+      45
+    ],
+    "1986-01-01 05:00": [
+      "-6:00/-5:00",
+      54
+    ]
+  },
+  "ecgye": {
+    "1970-01-01 00:00": [
+      "-5:00/-4:00",
+      45
+    ]
+  },
+  "eetll": {
+    "1970-01-01 00:00": [
+      "+3:00/+4:00",
+      95
+    ],
+    "1989-03-25 23:00": [
+      "+2:00/+3:00",
+      47
+    ]
+  },
+  "egcai": {
+    "1970-01-01 00:00": [
+      "+2:00/+3:00",
+      47
+    ]
+  },
+  "eheai": {
+    "1970-01-01 00:00": [
+      "-1:00",
+      null
+    ],
+    "1976-04-14 01:00": [
+      "+0:00/+1:00",
+      48
+    ],
+    "2018-10-28 02:00": [
+      "+0:00/+1:00",
+      null
+    ]
+  },
+  "erasm": {
+    "1970-01-01 00:00": [
+      "+3:00",
+      4
+    ]
+  },
+  "esceu": {
+    "1970-01-01 00:00": [
+      "+0:00/+1:00",
+      48
+    ],
+    "1984-03-16 00:00": [
+      "+1:00/+2:00",
+      46
+    ]
+  },
+  "eslpa": {
+    "1970-01-01 00:00": [
+      "+0:00/+1:00",
+      48
+    ]
+  },
+  "esmad": {
+    "1970-01-01 00:00": [
+      "+1:00/+2:00",
+      46
+    ]
+  },
+  "etadd": {
+    "1970-01-01 00:00": [
+      "+3:00",
+      4
+    ]
+  },
+  "fihel": {
+    "1970-01-01 00:00": [
+      "+2:00/+3:00",
+      47
+    ]
+  },
+  "fimhq": {
+    "1970-01-01 00:00": [
+      "+2:00/+3:00",
+      47
+    ]
+  },
+  "fjsuv": {
+    "1970-01-01 00:00": [
+      "+12:00/+13:00",
+      50
+    ]
+  },
+  "fkpsy": {
+    "1970-01-01 00:00": [
+      "-4:00",
+      49
+    ],
+    "1983-05-01 04:00": [
+      "-3:00/-2:00",
+      49
+    ],
+    "1985-09-15 03:00": [
+      "-4:00/-3:00",
+      49
+    ],
+    "2010-09-05 06:00": [
+      "-3:00",
+      49
+    ]
+  },
+  "fmksa": {
+    "1970-01-01 00:00": [
+      "+12:00",
+      81
+    ],
+    "1998-12-31 12:00": [
+      "+11:00",
+      81
+    ]
+  },
+  "fmpni": {
+    "1970-01-01 00:00": [
+      "+11:00",
+      116
+    ]
+  },
+  "fmtkk": {
+    "1970-01-01 00:00": [
+      "+10:00",
+      134
+    ]
+  },
+  "fotho": {
+    "1970-01-01 00:00": [
+      "+0:00/+1:00",
+      48
+    ]
+  },
+  "frpar": {
+    "1970-01-01 00:00": [
+      "+1:00/+2:00",
+      46
+    ]
+  },
+  "galbv": {
+    "1970-01-01 00:00": [
+      "+1:00",
+      6
+    ]
+  },
+  "gazastrp": {
+    "1970-01-01 00:00": [
+      "+2:00/+3:00",
+      74
+    ],
+    "1995-12-31 22:00": [
+      "+2:00/+3:00",
+      47
+    ]
+  },
+  "gblon": {
+    "1970-01-01 00:00": [
+      "+1:00",
+      null
+    ],
+    "1971-10-31 02:00": [
+      "+0:00/+1:00",
+      53
+    ]
+  },
+  "gdgnd": {
+    "1970-01-01 00:00": [
+      "-4:00",
+      18
+    ]
+  },
+  "getbs": {
+    "1970-01-01 00:00": [
+      "+4:00/+5:00",
+      null
+    ],
+    "1991-03-30 22:00": [
+      "+3:00/+4:00",
+      null
+    ],
+    "1991-04-08 20:00": [
+      "+3:00/+4:00",
+      56
+    ],
+    "1994-09-24 20:00": [
```

Comment:
Observation: A time zone that adopted new offsets without a metazone, and then put itself in a metazone a week later

Reply:
Addressed in #6767